changelog.md

```diff
@@ -1,4 +1,4 @@
 ## ROC'n'ROLL
 
-### Version 0.1.1
```

Comment:
The historical logs should be left beneath the newest log (in this case 0.1.3). Pretty please, fill previously removed logs